<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1954</width>
    <height>1054</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelRGB1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRGB2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRGB3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRGB4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>500</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenBtn">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>20</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>open Camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRGB5">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>500</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>70</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>100</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>140</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>170</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_5">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>200</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>70</y>
      <width>41</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>exp1:

exp2:

exp3:

exp4:

exp5:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>240</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1954</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
